neuro-sdk/src/neuro_sdk/_jobs.py

```diff
@@ -1131,8 +1131,9 @@ def _job_telemetry_from_api(value: Dict[str, Any]) -> JobTelemetry:
 
 
 def _volume_to_api(volume: Volume, config: Config) -> Dict[str, Any]:
+    assert config.project_name, "Project is not configured"
```

Comment:
minor. maybe we should also raise RuntimeError here? AssertionError might be disabled if the interpreter is run with -p or something.